autovalet_navigation/config/husky/static_map_params.yaml

```diff
@@ -1,2 +0,0 @@
-map_topic: "/rtabmap/grid_map"
```

Comment:
why did you delete these files (same one in husky and sim dirs)? where does this info live now?

Reply:
file cleanup. Neater that way